### PR TITLE
Feature/AT16-54_apollo_service

### PR DESCRIPTION
**Pull request by alexcastro72577**

### CODE REVIEW

File: APOLLO_SERVICE/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+type PostsResult {
+    success: Boolean!
+    errors: [String]
+    posts: [Post]
+}
+
+type Query {
+    listPosts: PostsResult!
+    getPost(id: ID!): PostResult!
+}
```
Comment (ajurado91) [inline APOLLO_SERVICE/schema.graphql:27]: Blank end line missed

File: APOLLO_SERVICE/api/queries.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# All rights reserved.
+#
+# This software is the confidential and proprietary information of
+# Jalasoft, ("Confidential Information"). You shall # not
+# disclose such Confidential Information and shall use it only in
+# accordance with the terms of the license agreement you entered into
+# with Jalasoft.
+#
+from .models import Post
+def listPosts_resolver(obj, info):
```
Comment (ajurado91) [inline APOLLO_SERVICE/api/queries.py:18]: Two blank lines, PEP 8

File: APOLLO_SERVICE/api/models.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    created_at = db.Column(db.Date)
+    def to_dict(self):
+        return {
+            "id": self.id,
+            "title": self.title,
+            "description": self.description,
+            "created_at": str(self.created_at.strftime('%d-%m-%Y'))
+        }
```
Comment (ajurado91) [inline APOLLO_SERVICE/api/models.py:26]: Blank end line missed

File: APOLLO_SERVICE/api/models.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+# All rights reserved.
+#
+# This software is the confidential and proprietary information of
+# Jalasoft, ("Confidential Information"). You shall # not
+# disclose such Confidential Information and shall use it only in
+# accordance with the terms of the license agreement you entered into
+# with Jalasoft.
+#
+from app import db
+
```
Comment (ajurado91) [inline APOLLO_SERVICE/api/models.py]: Two blank lines, PEP 8

File: APOLLO_SERVICE/app.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+# with Jalasoft.
+#
+from api import app, db
+from ariadne import load_schema_from_path, make_executable_schema, \
+    graphql_sync, snake_case_fallback_resolvers, ObjectType
+from ariadne.constants import PLAYGROUND_HTML
+from flask import request, jsonify
+from api.queries import listPosts_resolver, getPost_resolver
+from api.mutations import create_post_resolver, update_post_resolver, delete_post_resolver
+
```
Comment (Amilcar-B) [inline APOLLO_SERVICE/app.py:20]: must have two space

**Pull request closed without merging**